controllers/extensions/addon_controller_stages.go

```diff
@@ -261,6 +261,14 @@ func (r *deletionStage) Handle(ctx context.Context) {
 
 func (r *installableCheckStage) Handle(ctx context.Context) {
 	r.process(func(addon *extensionsv1alpha1.Addon) {
+		// additional check to the addon YAML to ensure support for Kubernetes versions prior to 1.2.5
```

Comment:
The comment should explain why an additional check is needed prior to K8s 1.25, and the `1.2.5` should be 1.25